### PR TITLE
Certificates files elaboration in docs

### PR DESCRIPTION
**Pull request by teameh**
This was missing, took me while to find it in the changelogs..

### CODE REVIEW

**Comment by teameh** [comment]
Same in `BoltKernelExtension.java` btw

**Comment by zhenlineo** [comment]
Sorry to come back to your PR so late. Do you also mind to sign [CLA](https://neo4j.com/developer/cla/) so that we could merge you PR?

**Comment by teameh** [comment]
Haha just for adding: "Certificates files must be named neo4j.cert and neo4j.key" ?

Done. See my email.

**Comment by zhenlineo** [comment]
Thanks a lot. You are also welcome to add more of course!
We will merge the PR after it passes all tests later.
Thanks again for fixing the docs.

File: community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java [hunk 1]
```diff
@@ -122,8 +122,9 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
             },
             EMPTY );
 
-    @Description( "Directory for storing certificates to be used by Neo4j for TLS connections" )
-    Setting<File> certificates_directory = BoltKernelExtension.Settings.certificates_directory;
+    @Description( "Directory for storing certificates to be used by Neo4j for TLS connections. Certificates files must be named neo4j.cert and neo4j.key" )
+    Setting<File>
```
Comment (jjaderberg) [inline community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java]: Where did the field name `certificates_directory` go?

File: community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java [hunk 2]
```diff
@@ -122,8 +122,9 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
             },
             EMPTY );
 
-    @Description( "Directory for storing certificates to be used by Neo4j for TLS connections" )
-    Setting<File> certificates_directory = BoltKernelExtension.Settings.certificates_directory;
+    @Description( "Directory for storing certificates to be used by Neo4j for TLS connections. Certificates files must be named neo4j.cert and neo4j.key" )
```
Comment (jjaderberg) [inline community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java]: `Certificates files` => `Certificate files`
Files and paths with italics (surrounded by underscore):
`neo4j.cert` => `_neo4j.cert_`
`neo4j.key` => `_neo4j.key_`

**Review by jjaderberg — CHANGES_REQUESTED** [review]

**Comment by teameh** [comment]
@jjaderberg whoops.. dunno what went wrong there. Fixed

**Comment by teameh** [comment]
You want me to retarget to 3.2 btw?

**Review by jjaderberg — APPROVED** [review]

**Pull request merged**